packages/next-dashboard/sass/components/_select.scss

```diff
@@ -176,3 +176,8 @@
     }
   }
 }
+.header-content-container {
+  .select__value-container {
+    padding: 10px 15px;
+  }
+}
```

Comment:
Since the rest of the icon code is defined in the pier project, should this really go into the next-dashboard project?

Reply:
Yeah I think so, but maybe it's better to move this to the pier project. Not sure what the best call is here.

these styles where removed here, but they're needed for making the searchbar look good.
https://github.com/PiJaAB/next-dashboard/commit/bc952c463264ac5527c4c4511be4cf919449d023#diff-139b248d92a9a4ebcca27a7373266c4fL22

Reply:
Let's wait for input from @FireyFly maybe, personally I think adding this to the Pier project, if all other companion code is for that one would be the best call, since this specific style of searchbar seems rather specific to that project

Reply:
Yeah, maybe we should move the other styles search styles from the next-dashboard project to the pier project also then. (`_search.scss`)

Reply:
I think it makes sense to keep styling & assets together with the JS code unless there's technical reasons prohibiting that. i.e., if the searchbar and its styling is specific to that project (for the time being at least), let's move the styling to that project!

If it's desirable to change that in the future, we can always migrate code back, but then we should do that as one unit and not e.g. only the style or implementation.